### ISSUE TITLE
LHN on HybridApp shows chats as Hidden and on web just empty spaces between the commas

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#53891]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:**
**Reproducible in staging?:** Needs Reproduction
**Reproducible in production?:** Needs Reproduction
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:**
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @davidcardoza 
**Slack conversation** (hyperlinked to channel name): [expensify_quality](https://expensify.slack.com/archives/C05LX9D6E07/p1733858044771659)

## Action Performed:

1. Signout/Signin on HybridApp
2. Navigate to the left-hand navigation (LHN) panel.

## Expected Result:
Chats showing the email of the user

## Actual Result:
On Hybrid app shows chat as Hidded
On web  just empty spaces between the commas

## Workaround:
Unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [x] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

<details>
  <summary>Add any screenshot/video evidence</summary>

  
</details>

<img src="https://github.com/user-attachments/assets/0bede379-75c5-401e-98b4-6af857dc99d6" width="40%" height="40%">

<img src="https://github.com/user-attachments/assets/f25c9a2b-5e30-45bb-ab02-85c124d01c30" width="40%" height="40%">

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Issue Owner</summary>Current Issue Owner: @MarioExpensify</details>

**Comment by MelvinBot** [comment]
This has been labelled "Needs Reproduction". Follow the steps here: https://stackoverflowteams.com/c/expensify/questions/16989

**Comment by MarioExpensify** [comment]
Checking

**Comment by MarioExpensify** [comment]
Requested QA team to check if they can reproduce it [here](https://expensify.slack.com/archives/C9YU7BX5M/p1734435244393469)

**Comment by muttmuure** [comment]
We need to get the Onyx data for these reports in the thread please

**Comment by muttmuure** [comment]
Doza should be able to right click one and copy Onyx data

**Comment by twisterdotcom** [comment]
I agree, I can't recreate.

**Comment by davidcardoza** [comment]
Sorry for the delay, I was OOO yesterday. I can no longer reproduce. I will reopen the issue if/when it sprouts up again.

**Comment by davidcardoza** [comment]
This issue has been reproduced by @zsgreenwald 
- Slack conversation - https://expensify.slack.com/archives/C05LX9D6E07/p1734546191501819
- Onyx state - [onyx-state.txt](https://github.com/user-attachments/files/18189572/onyx-state.txt)

**Comment by muttmuure** [comment]
Sorry to delete that, let's just err on the side of caution with posting the onyx state to GH. We redact sensitive information but I would rather we kept them in Slack just to be 100% safe

**Comment by davidcardoza** [comment]
@muttmuure - Sounds good, am I good to just link to the Slack convo in that case?

**Comment by muttmuure** [comment]
Yeah Slack convo is great!

**Comment by davidcardoza** [comment]
[- Slack conversation](https://expensify.slack.com/archives/C05LX9D6E07/p1734581843910439)

**Comment by MarioExpensify** [comment]
Hey, thanks for the Onyx data, I'm wrapping up another issue and will come back to this soon. @muttmuure this is the first time I'll be working on Onyx related stuff, I'm may need some assistance 😅

**Comment by MarioExpensify** [comment]
Sorry for the late update, yesterday I worked on this one and I'm still trying to pinpoint where the "Hidden" was introduced in the pipeline. I'll bring updates today.

**Comment by MarioExpensify** [comment]
I've left an update here, we've got an Onyx state export and it seems the personalDetailsList is missing a lot of data (that is the reason we're seeing the "Hidden" labels). I'm checking where we're sending the onyx updates that are possibly overwriting the valid values with empty. But so far it seems to be a backend issue.

**Comment by MarioExpensify** [comment]
As discussed [here](https://expensify.slack.com/archives/C05LX9D6E07/p1735683351514899?thread_ts=1733858044.771659&cid=C05LX9D6E07), we're moving on removing the known user checks for expensify employees. PR is already drafted, I'll just wait on a confirmation before moving forward.

**Comment by MarioExpensify** [comment]
From @zsgreenwald comment [here](https://expensify.slack.com/archives/C05LX9D6E07/p1734546191501819) the bug may have happened again after we deployed the fix. I'll leave this issue open for another week so we can observe.

**Comment by MarioExpensify** [comment]
Okay, haven't received any more comments on this. Will be closing it, we can reopen if it surfaces back.

**Comment by davidcardoza** [comment]
I think we are experiencing the same issue again here https://expensify.enterprise.slack.com/archives/C05LX9D6E07/p1736985872463399

**Comment by MarioExpensify** [comment]
Trying to grab the exported Onyx State from someone affected by the issue so we can investigate if our previous fix at least changed the behavior.

**Comment by MarioExpensify** [comment]
Still investigating

**Comment by muttmuure** [comment]
Sounds like we can in fact close this again. Let's do that
